src/traverse.js

```diff
@@ -30,5 +31,9 @@ var sequence = require('./sequence');
  *      R.traverse(Maybe.of, safeDiv(10), [2, 0, 5]); //=> Nothing
  */
 module.exports = _curry3(function traverse(of, f, traversable) {
-  return sequence(of, map(f, traversable));
+  return typeof traversable['fantasy-land/traverse'] === 'function' ?
+    traversable['fantasy-land/traverse'](f, of) :
+    reduceRight(function(acc, x) { return ap(map(prepend, f(x)), acc); },
+                of([]),
+                traversable);
 });
```

Comment:
Could someone review this change? I copied the existing `sequence` implementation but replaced `x` with `f(x)` in `ap(map(prepend, x), acc)`. Is that correct?